lib/geolocation.js

```diff
@@ -70,10 +72,14 @@ async function setLocationWithIdb (idb, latitude, longitude) {
  * @throws {Error} If it failed to set the location
  */
 async function setLocationWithAppleScript (sim, latitude, longitude, menu = 'Debug') {
-  const decimalSeparator = (0.1).toLocaleString().replace(/\d/g, '');
   // Make sure system-wide decimal separator is used
-  const latitudeStr = `${latitude}`.replace(/\D/g, decimalSeparator);
-  const longitudeStr = `${longitude}`.replace(/\D/g, decimalSeparator);
+  const decimalSeparator = (await sim.executeUIClientScript(`
```

Comment:
this might cause security alert. Try to use `exec` with `/usr/bin/python -c "from AppKit import NSNumberFormatter;import sys; sys.stdout.write(NSNumberFormatter.alloc().init().decimalSeparator())"` command

Reply:
Can we test whether or not it does? Had no issues here on latest macOS.
Feel like we should be very careful here about introducing a `python` dependency.

Reply:
python is a part of macOS (at least the one under `/usr/bin`). This command does exactly the same that osascript does, just without any overhead

Reply:
good to know, thanks!